demo/index.js

```diff
@@ -232,7 +232,7 @@ import { OpenSheetMusicDisplay } from '../src/OpenSheetMusicDisplay/OpenSheetMus
             function() {
                 return onLoadingEnd(isCustom);
             }, function(e) {
-                console.warn(e.stack);
+                console.warn("Error rendering sheet: " + e + " StackTrace: \n" + e.stack);
```

Comment:
e.stack can be undefined and e.toString can contain important information (UnformattedNoteError from VexFlow when calling getBoundingBox())